src/config/currencies.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use PodPoint\I18n\CurrencyCode;
+
+return [
+    CurrencyCode::POUND_STERLING => [
+        'precision' => 2,
+    ],
+    CurrencyCode::EURO => [
+        'precision' => 2,
+    ],
+    CurrencyCode::NORWEGIAN_KRONE => [
+        'precision' => 2,
+    ],
```

Comment:
are we sure on this values?

Reply:
Your comment made me think that we don't need this config file actually. I'm pushing the new version